pages/plugins/openapi.md

```diff
@@ -124,14 +163,27 @@ public class MyApplication {
       app.post("/users", OpenApiBuilder.documented(createUserDocumentation, ctx -> {
           // ...
       }));
-  }
+  }https://docs.oracle.com/javase/7/docs/technotes/guides/language/annotations.html
```

Comment:
This looks a bit out of place.

Reply:
rogue url ... just checking if someone else finds bugs ... there has to be always at least one bug. :smirk: